examples/pex/sample-python-script/sample_python_script/main.py

```diff
@@ -7,7 +7,6 @@
 
 @click.command()
 @click.option("--is-data/--no-is-data", default=False, help="Whether or not this is run as a data source")
-@click.option("--is-delete/--no-is-delete", default=False, help="Whether or not this is run as a delete provisioner")
 @click.option("--triggers-json", help="JSON encoded triggers for the resource")
 def main(is_data, is_delete, triggers_json):
```

Comment:
You need to remove `is_delete` here (I think that is what is causing the exception in test)

Reply:
Haha, just caught that 👍 

https://github.com/gruntwork-io/package-terraform-utilities/pull/30/commits/61b19f3eafebca94e78049629e106675421c3605